pkg/apis/batch/validation/validation_test.go

```diff
@@ -921,22 +921,6 @@ func TestValidateJobUpdate(t *testing.T) {
 				job.Spec.ManualSelector = pointer.BoolPtr(true)
 			},
 		},
-		"immutable completion": {
```

Comment:
I think this case should remain, since this is a non-indexed job, which the KEP didn't talk about relaxing validation for

Reply:
Makes sense, added it back under name "immutable completions for non-indexed jobs"